include/pstore/mcrepo/compilation.hpp

```diff
@@ -107,7 +107,7 @@ namespace pstore {
             /// The extent of the fragment referenced by this compilation symbol.
             extent<fragment> fext;
             typed_address<indirect_string> name;
-            linkage_type linkage;
+            repo::linkage linkage;
             visibility_type visibility;
```

Comment:
Should we change `visibility_type` to `visibility`?

Reply:
Yes, that's up next. Because the change hits multiple repositories, I'm taking it one at a time.